cmd/base.go

```diff
@@ -103,15 +103,39 @@ func parseConfig() (*bc.Config, error) {
 }
 
 func EnsureCLIFlags(cmd *cobra.Command, appcfg *bc.Config) {
-
 	assignFields := func(p reflect.Type, elem reflect.Value, name string) {
 		for i := 0; i < p.NumField(); i++ {
 			if p.Field(i).Tag.Get("mapstructure") == name {
-				elem.Field(i).Set(reflect.ValueOf(viper.Get(name)))
+				var val interface{}
```

Comment:
why did you do this? does it fix the bug where we couldn't load types that are not `int` (for numbers, i.e : uint64 won't work)

Reply:
Yes. `spacemeshos/post` config introduced `uint64` type so it was necessary.

Reply:
can you add support for uint16? it is necessary in other configs

Reply:
Not supported by viper :\
All supported types were added.